docs/requirements.txt

```diff
@@ -1,10 +1,16 @@
 # ReadTheDocs uses the `environment.yaml` so make sure to update that as well
 # if you change this file
```

Comment:
Shouldn't we remove this comment because we now don't use conda any more?

Reply:
Oh, right. I'll remove it now.